src/components/AllIngredients/AllIngredients.tsx

```diff
@@ -152,6 +152,21 @@ function AllIngredients({ backend, user }: AllIngredientsProps) {
 		ingredient.name.toLowerCase().includes(searchQuery.toLowerCase())
 	);
 
+    const highlightText = (text: string, query: string) => {
+        if (!query) return text;
+    
+        const parts = text.split(new RegExp(`(${query})`, "gi"));
+        return parts.map((part, index) =>
+          part.toLowerCase() === query.toLowerCase() ? (
+            <span key={index} style={{ backgroundColor: "yellow", fontWeight: "bold" }}>
```

Comment:
Would you mind if I ask you to make the text bold but not yellowly highlighted?
Also, since now we have the padding all around, could you align the "CREATE" button with the table (with the left padding) please?
![image](https://github.com/user-attachments/assets/7ad98223-b0e5-4c41-a8d7-ddd995f72717)